src/App.tsx

```diff
@@ -1,13 +1,41 @@
 import { Route, Routes } from 'react-router-dom';
 import { Footer } from './templates/Footer/Footer';
+import { Header } from './templates/Header/Header';
+import { HeaderProps } from './templates/Header/Header.types';
+import { useMediaQuery } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import { useTranslation } from 'react-i18next';
 
 export function App() {
+  const { t } = useTranslation();
+  const theme = useTheme();
+  const tablet = useMediaQuery(theme.breakpoints.up('tablet'));
+  /* simulating the separation of the login/register page from the header component */
+  /* this will be based on breakpoints and the page later on*/
+
+  const TABLET_IDX_ADJUST = 0;
+  const MOBILE_IDX_ADJUST = 1;
+
+  const headerTabletJsonIdx = tablet ? TABLET_IDX_ADJUST : MOBILE_IDX_ADJUST;
+  const jsonIdx = headerTabletJsonIdx;
+  const jsonSubscript = `.${String(jsonIdx)}`;
+
+  const headerProps: HeaderProps = {
+    dormitory: { url: t(`header.buildingUrl`), urlName: t('header.buildingName') },
+    faculty: { url: t('header.facultyUrl'), urlName: t('header.faculty') },
+    headerTitle: t(`loginForm.loginTitle${jsonSubscript}`),
+  };
+
```

Comment:
do not simulate this logic as this doesn't make any sense, use only login or register and later add translations for other page. When you will be working on #17 or #18

Reply:
should be fixed in 758defa